brightcove-exoplayer/README.md

```diff
@@ -125,7 +125,11 @@ To explain in more detail:
  1. Extends *MainActivity* to use the *BrightcovePlayer* class, which handles activity lifecycle behavior for the Brightcove player used.
  2. Instantiates the BrightcoveExoPlayerVideoView from the layout XML set with *setContentView()*.
  3. Loads a sample video from the Media API of Brightcove VideoCloud, given an authorization token and a video id.
- 4. A XML declaration for the BrightcoveExoPlayerVideoView from the activity's layout file.
+ 4. An XML declaration for the BrightcoveExoPlayerVideoView from the activity's layout file.
+
+## Known Issues
```

Comment:
I'm not sure this is the right place.  Is there a release-notes.txt file?  That's where I would be inclined to put it, creating one if it does not already exist.